content/sql/concepts/commands/terms/contains/contains.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'CONTAINS'
+Description: 'Searches for matches to words or phrases.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+CONTAINS() is a predicate used to search for matches to words or phrases in a column.
+
+## Code Example
+
+`CONTAINS()` takes two arguements. A column, and the word/phrase to search for.
```

Comment:
```suggestion
`CONTAINS()` takes two arguments. A column, and the word or phrase to search for.
```

Reply:
Let's move this explanation underneath the syntax block below! Notice I've suggested a heading change here to "Syntax"